src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        int number = new Random().nextInt(99) + 1;
+        return new Ball(new ColorSupplier().getRandomColor(), number);
+    }
+
+}
```

Comment:
1. Magic numbers should be constants with proper naming
2. Services we rely on should be class variables in order not to create new object each time method is called
3. Pls remove unnecessary empty line
```suggestion
public class Lottery {
    private static final int NUMBER_LIMIT = 100;
    
    private final Random random = new Random();
    private final ColorSupplier colorSupplier = new ColorSupplier();

    public Ball getRandomBall() {
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(NUMBER_LIMIT));
    }
}
```